### ISSUE TITLE
BuildEventContext.ProjectInstanceId and BuildEventContext.BuildRequestId should be exposed from the TimedNode model objects

### ISSUE THREAD

**Issue by MarkKharitonov** [KirillOsenkov/MSBuildStructuredLog#553]
Suppose I would like to build the trace view of the project build events using the approach of https://github.com/rainersigwald/TraceEventLogger/blob/master/TraceEventLogger/TraceEventLogger.cs.

In this case we need 3 properties from `BuildEventContext`:

1. NodeId - already exposed
2. ProjectInstanceId
3. BuildRequestId

**Comment by KirillOsenkov** [comment]
Did you see the new horizontal Tracing view added in 2.1.596?

https://twitter.com/KirillOsenkov/status/1475562239901925378

**Comment by MarkKharitonov** [comment]
Yes, I did and it looks very promising. But it is the first version and it is likely to take time until it matures.

In the meantime I think it is useful to allow 3rd party to build custom tracing logic.
